package.json

```diff
@@ -158,11 +173,14 @@
     "react-virtualized": "^9.22.3",
     "redux": "^3.5.0",
     "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.3.0",
+    "redux-thunk": "^2.2.0",
```

Comment:
I had to downgrade a minor version because of a typing issue https://github.com/reduxjs/redux-thunk/issues/205